### DISCUSSION TITLE
MSE and OpenGL...

### DISCUSSION THREAD

**Discussion by fredvs** [mse-org/mseide-msegui#53] [General]
I just discover a new hidden gem: `mseopenglwidget`:

The hidden demo is in https://github.com/mse-org/mseuniverse:

`/mseuniverse/attic/msedocumenting/mse/trunk/help/tutorials/widgets/opengl/openglwidget/openglwidget.prj
`

The anim is incredibly fast, even in full screen:

![Capture d’écran_2020-09-19_23-22-03](https://user-images.githubusercontent.com/3421249/93689557-fe4f3700-facf-11ea-8a92-ed337d4dee44.png)

**Comment by funlw65** [comment]
Thank you for the pointer!
